docs/index.md

```diff
@@ -90,13 +90,13 @@ FastAPI stands on the shoulders of giants:
 ## Installation
 
 ```bash
-$ pip install fastapi
```

Comment:
Hey 🤔  What is you thoughts on using `$`? It seems different compared to the rest of the docs.